pkg/apis/extensions/validation/validation.go

```diff
@@ -810,6 +812,18 @@ func validatePSPFSGroup(fldPath *field.Path, groupOptions *extensions.FSGroupStr
 	return allErrs
 }
 
+// validatePSPAllowedHostPath validates the allowed host paths of the PodSecurityPolicy.
+func validatePSPAllowedHostPath(fldPath *field.Path, paths []string) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	for _, path := range paths {
+		if !filepath.IsAbs(path) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("allowedHostPaths"), path, "must specify a absolute path"))
```

Comment:
> field.Invalid(fldPath.Child("allowedHostPaths")

1. you've already specified a child in the argument to this function (`validatePSPAllowedHostPath(fldPath.Child("allowedHostPaths")`) and you shouldn't invoke `Child` again because it leads to a wrong path (`allowedHostPaths.allowedHostPaths`).

2. I'm suggesting to use `Index()` here to complain on exact element of the slice (see [example](https://github.com/kubernetes/kubernetes/blob/45b7f5a4b0f0de312ef88cdc802ee574806b2a6e/pkg/apis/extensions/validation/validation.go#L825-L827)). (And in ideal is to write a test for that case.)